catalog/registry/catalog-registry-cswinputtransformer/src/test/java/ddf/catalog/registry/transformer/RegistryTransformerTest.java

```diff
@@ -129,7 +129,7 @@ public void testBasicInfo() throws Exception {
         RegistryMetacardImpl meta = convert("/csw-basic-info.xml");
         assertRegistryMetacard(meta);
 
-        assertThat(meta.getId(), is("2014ca7f59ac46f495e32b4a67a51276"));
```

Comment:
Since we're no longer setting the id member in the converter, should we be asserting that getId() returns some other value?